packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -18,6 +18,7 @@ const selectors = {
 	// Within the editor body.
 	blockAppender: '.block-editor-default-block-appender',
 	paragraphBlocks: 'p.block-editor-rich-text__editable',
+	blockWarning: '.block-editor-warning',
```

Comment:
Leaving this class based because there could hypothetically be different warnings in here over time